modules/internal/localeModels/flat/LocaleModel.chpl

```diff
@@ -312,11 +312,11 @@ module LocaleModel {
   //
   // runtime interface
   //
-  extern proc chpl_comm_fork(loc_id: int, subloc_id: int,
+  extern proc chpl_comm_execute_on(loc_id: int, subloc_id: int,
                              fn: int, args: c_void_ptr, arg_size: size_t);
```

Comment:
The argument list on the 2nd line should be re-indented to align with the beginning of the args on the 1st line.